src/model/entities/status-message.ts

```diff
@@ -0,0 +1,9 @@
+import { StatusCode } from '../enums/status-code'
+
+export interface StatusMessage {
+  id: string
+  statusCode: StatusCode
+  title: string
+  message: string
+  lastUpdatedTimestamp?: number
```

Comment:
Just cause OCD i guess 😆 

```suggestion
  id: string
  title: string
  message: string
  statusCode: StatusCode
  lastUpdatedTimestamp?: number
```

Reply:
Updated